.gitattributes

```diff
@@ -1,3 +1,50 @@
-tests/** export-ignore
-tests/ export-ignore
-chart/tests export-ignore
+airflow/providers export-ignore
+breeze export-ignore
+breeze-legacy export-ignore
+breeze-complete export-ignore
+chart export-ignore
```

Comment:
Did you test this for the chart release steps? The chart also uses `git archive`.

Reply:
good point

Reply:
Very good call. 
Thanks Jed. It affected chart. I had to remove providers too.